lib/commands/geolocation.js

```diff
@@ -36,16 +36,30 @@ export async function setGeoLocation(location) {
 }
 
 /**
+ * Set the device geolocation.
+ *
  * @this {import('../driver').AndroidDriver}
- * @param {number} latitude
- * @param {number} longitude
- * @param {number} [altitude]
+ * @param {number} latitude Valid latitude value.
+ * @param {number} longitude Valid longitude value.
+ * @param {number} [altitude] Valid altitude value.
+ * @param {number} [satellites] Number of satellites being tracked (1-12).
```

Comment:
we should also mention which params are only applicable for emulators and which for real devices